resources/zib/zib-Procedure-ServiceRequest.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-Procedure-ServiceRequest" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-Procedure-ServiceRequest" />
+  <name value="ZibProcedureServiceRequest" />
+  <title value="zib Procedure ServiceRequest" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This profile covers the recording of therapeutic or diagnostic procedure that the patient will undergo. The Procedure profile (http://nictiz.nl/fhir/StructureDefinition/zib-Procedure) covers the procedure undergone by the patient." />
+  <purpose value="This ServiceRequest resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Procedure](https://zibs.nl/wiki/Procedure-v5.2(2020EN))." />

```

Comment:
Not conform the profiling guidelines:

This [resource type] resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [English zib name] version]([link to the English zib page on zibs.nl]).

Reply:
Fixed!